docs/resources/athenz_role.md

```diff
@@ -44,6 +44,10 @@ resource "athenz_role" "foo_role" {
     name = "domain3.user3"
     review = "2023-12-29 23:59:59"
   }
+  settings {
```

Comment:
It should be explained in the section "Argument Reference"

Reply:
added to the docs